packages/manager/modules/core/src/manager-core.constants.js

```diff
@@ -1,83 +1,3 @@
-export const REDIRECT_URLS = {
-  support: 'https://www.ovh.com/manager/dedicated/index.html#/ticket',
-  billing: 'https://www.ovh.com/manager/dedicated/index.html#/billing/history',
-  billingPayments:
-    'https://www.ovh.com/manager/dedicated/index.html#/billing/payments',
-  billingMean: 'https://www.ovh.com/manager/dedicated/index.html#/billing/mean',
-  billingVouchers:
-    'https://www.ovh.com/manager/dedicated/index.html#/billing/vouchers',
-  billingRefunds:
-    'https://www.ovh.com/manager/dedicated/index.html#/billing/refunds',
-  billingFidelity:
-    'https://www.ovh.com/manager/dedicated/index.html#/billing/fidelity',
-  billingCredits:
-    'https://www.ovh.com/manager/dedicated/index.html#/billing/credits',
-  ordersInProgress:
-    'https://www.ovh.com/manager/dedicated/index.html#/billing/orders?status=in-progress',
-  orders:
-    'https://www.ovh.com/manager/dedicated/index.html#/billing/orders?status=all',
-  services:
-    'https://www.ovh.com/manager/dedicated/index.html#/billing/autoRenew',
-  servicesAgreements:
-    'https://www.ovh.com/manager/dedicated/index.html#/useraccount/agreements',
-  paymentMeans:
-    'https://www.ovh.com/manager/dedicated/index.html#/billing/mean',
-  addCreditCard:
-    'https://www.ovh.com/manager/dedicated/index.html#/billing/mean/add?meanType=creditCard',
-  ovhAccount:
-    'https://www.ovh.com/manager/dedicated/index.html#/billing/ovhaccount',
-  debtAccount:
-    'https://www.ovh.com/manager/dedicated/index.html#/billing/statements',
-  userInfos:
-    'https://www.ovh.com/manager/dedicated/index.html#/useraccount/infos',
-  userSecurity:
-    'https://www.ovh.com/manager/dedicated/index.html#/useraccount/security',
-  userEmails:
-    'https://www.ovh.com/manager/dedicated/index.html#/useraccount/emails',
-  userSubscriptions:
-    'https://www.ovh.com/manager/dedicated/index.html#/useraccount/subscriptions',
-  userSSH: 'https://www.ovh.com/manager/dedicated/index.html#/useraccount/ssh',
-  userAdvanced:
-    'https://www.ovh.com/manager/dedicated/index.html#/useraccount/advanced',
-  contacts:
-    'https://www.ovh.com/manager/dedicated/index.html#/useraccount/contacts?tab=SERVICES&serviceName={serviceName}',
-  horizon:
-    'https://horizon.cloud.ovh.net/openstackdashboard?username={username}',
```

Comment:
I'm wondering if removing this particular URL still work as expected!

Reply:
Thanks to report it! 
This one is not used. There is some other `REDIRECT_URLS` constants (with `horizon` inside) and it's done in another PR: https://github.com/ovh/manager/pull/4197/files#diff-206c4f156a8a305eec45c2b281664dc31fa64174e988507b1baac8b57c3e6038R1-R9